### PR TITLE
Update files with PEP8 standards in folder dojo #002

### PR DESCRIPTION
**Pull request by ajtortolero**
Update files with PEP8 standars in folders dojo/

Folder affected
dojo/announcement
dojo/announcement
dojo/api_v2
dojo/authorization
dojo/banner
dojo/benchmark
dojo/components

### CODE REVIEW

File: dojo/benchmark/views.py [hunk 1]
```diff
@@ -28,71 +39,79 @@ def add_benchmark(queryset, product):
 
     try:
         Benchmark_Product.objects.bulk_create(requirements)
-    except:
+    except BaseException:
```
Comment (Maffooch) [inline dojo/benchmark/views.py:42]:
```suggestion
    except Exception:
```

**Review by Maffooch — CHANGES_REQUESTED** [review]

File: dojo/benchmark/views.py (hunk 1 above)
Comment (ajtortolero) [reply dojo/benchmark/views.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8321/commits/5777783dc6f53ae0172a58eeb289b20c72f75997

**Review by Maffooch — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Review by devGregA — APPROVED** [review]

**Review by grendel513 — APPROVED** [review]

File: dojo/api_v2/views.py [hunk 2]
```diff
@@ -2482,50 +3528,56 @@ def report_generate(request, obj, options):
...
     report_info = "Generated By %s on %s" % (
-        user.get_full_name(), (timezone.now().strftime("%m/%d/%Y %I:%M%p %Z")))
+        user.get_full_name(),
+        (timezone.now().strftime("%m/%d/%Y %I:%M%p %Z")),
+    )
 
     # generate = "_generate" in request.GET
     report_name = str(obj)
-    report_type = type(obj).__name__
+    type(obj).__name__
```
Comment (cneill) [inline dojo/api_v2/views.py:3545]:
```suggestion

```

File: dojo/api_v2/views.py [hunk 3]
```diff
@@ -2482,50 +3528,56 @@ def report_generate(request, obj, options):
...
-
-        findings = ReportFindingFilter(request.GET, prod_type=product_type, queryset=prefetch_related_findings_for_report(Finding.objects.filter(
-            test__engagement__product__prod_type=product_type)))
-        products = Product.objects.filter(prod_type=product_type,
-                                          engagement__test__finding__in=findings.qs).distinct()
-        engagements = Engagement.objects.filter(product__prod_type=product_type,
-                                                test__finding__in=findings.qs).distinct()
-        tests = Test.objects.filter(engagement__product__prod_type=product_type,
-                                    finding__in=findings.qs).distinct()
+        str(product_type)
```
Comment (cneill) [inline dojo/api_v2/views.py:3556]:
```suggestion

```

File: dojo/api_v2/views.py [hunk 4]
```diff
@@ -2536,116 +3588,128 @@ def report_generate(request, obj, options):
...
 
     elif type(obj).__name__ == "Product":
         product = obj
 
         report_name = "Product Report: " + str(product)
-        report_title = "Product Report"
-        report_subtitle = str(product)
-        findings = ReportFindingFilter(request.GET, product=product, queryset=prefetch_related_findings_for_report(Finding.objects.filter(
-            test__engagement__product=product)))
+        str(product)
```
Comment (cneill) [inline dojo/api_v2/views.py:3604]:
```suggestion

```

File: dojo/api_v2/views.py [hunk 5]
```diff
@@ -2536,116 +3588,128 @@ def report_generate(request, obj, options):
...
+            engagement=engagement,
+            queryset=prefetch_related_findings_for_report(
+                Finding.objects.filter(test__engagement=engagement)
+            ),
+        )
         report_name = "Engagement Report: " + str(engagement)
 
-        report_title = "Engagement Report"
-        report_subtitle = str(engagement)
+        str(engagement)
```
Comment (cneill) [inline dojo/api_v2/views.py:3633]:
```suggestion

```

File: dojo/api_v2/views.py [hunk 6]
```diff
@@ -2536,116 +3588,128 @@ def report_generate(request, obj, options):
...
+            request.GET,
+            engagement=test.engagement,
+            queryset=prefetch_related_findings_for_report(
+                Finding.objects.filter(test=test)
+            ),
+        )
         report_name = "Test Report: " + str(test)
-        report_title = "Test Report"
-        report_subtitle = str(test)
+        str(test)
```
Comment (cneill) [inline dojo/api_v2/views.py:3652]:
```suggestion

```

File: dojo/benchmark/views.py [hunk 7]
```diff
@@ -130,78 +192,140 @@ def score_asvs(product, benchmark_type):
...
 
     # Create the benchmark summary category
     try:
-        benchmark_product_summary = Benchmark_Product_Summary.objects.get(product=product, benchmark_type=benchmark_type)
-    except:
-        benchmark_product_summary = Benchmark_Product_Summary(product=product, benchmark_type=benchmark_type)
+        benchmark_product_summary = Benchmark_Product_Summary.objects.get(
+            product=product, benchmark_type=benchmark_type
+        )
+    except BaseException:
```
Comment (cneill) [inline dojo/benchmark/views.py:218]:
```suggestion
    except Exception:
```

**Review by cneill — CHANGES_REQUESTED** [review]
Noticed a few places where phantom values are left behind, and one case of `BaseException` that should be `Exception`

File: dojo/api_v2/views.py (hunk 2 above)
Comment (ajtortolero) [reply dojo/api_v2/views.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8321/commits/3a2c9d288fe5aec61c4cc0739853f148a75e04bb

File: dojo/api_v2/views.py (hunk 4 above)
Comment (ajtortolero) [reply dojo/api_v2/views.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8321/commits/3a2c9d288fe5aec61c4cc0739853f148a75e04bb

File: dojo/api_v2/views.py (hunk 6 above)
Comment (ajtortolero) [reply dojo/api_v2/views.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8321/commits/3a2c9d288fe5aec61c4cc0739853f148a75e04bb

File: dojo/benchmark/views.py (hunk 7 above)
Comment (ajtortolero) [reply dojo/benchmark/views.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8321/commits/09e8746cdb2cce9fd1e93630862bc819b95e5c15

**Review by cneill — APPROVED** [review]

File: dojo/api_v2/views.py [hunk 8]
```diff
@@ -1059,28 +1509,33 @@ def get_duplicate_cluster(self, request, pk):
...
-        request_body=no_body
+        responses={status.HTTP_204_NO_CONTENT: ""}, request_body=no_body
+    )
+    @action(
+        detail=True, methods=["post"], url_path=r"original/(?P<new_fid>\d+)"
     )
-    @action(detail=True, methods=['post'], url_path=r'original/(?P<new_fid>\d+)')
     def set_finding_as_original(self, request, pk, new_fid):
-        finding = self.get_object()
+        self.get_object()
```
Comment (Maffooch) [inline dojo/api_v2/views.py:1538]: Is there a good reason to call this without using it?

File: dojo/api_v2/views.py [hunk 9]
```diff
@@ -1059,28 +1509,33 @@ def get_duplicate_cluster(self, request, pk):
         request_body=no_body,
         responses={status.HTTP_204_NO_CONTENT: ""},
     )
-    @action(detail=True, methods=['post'], url_path=r'duplicate/reset')
+    @action(detail=True, methods=["post"], url_path=r"duplicate/reset")
     def reset_finding_duplicate_status(self, request, pk):
-        finding = self.get_object()
-        checked_duplicate_id = reset_finding_duplicate_status_internal(request.user, pk)
+        self.get_object()
```
Comment (Maffooch) [inline dojo/api_v2/views.py:1514]: This one too

**Review by Maffooch — CHANGES_REQUESTED** [review]

File: dojo/api_v2/views.py (hunk 5 above)
Comment (ajtortolero) [reply dojo/api_v2/views.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/commit/09e8746cdb2cce9fd1e93630862bc819b95e5c15

File: dojo/api_v2/views.py (hunk 3 above)
Comment (ajtortolero) [reply dojo/api_v2/views.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/commit/09e8746cdb2cce9fd1e93630862bc819b95e5c15

File: dojo/api_v2/views.py (hunk 9 above)
Comment (ajtortolero) [reply dojo/api_v2/views.py]: Sorry, changes apply this commit https://github.com/DefectDojo/django-DefectDojo/pull/8321/commits/48d3037e3c39c1db66a2ca449728c2991ec19fc2

File: dojo/api_v2/views.py (hunk 8 above)
Comment (ajtortolero) [reply dojo/api_v2/views.py]: Sorry, changes apply this commit https://github.com/DefectDojo/django-DefectDojo/pull/8321/commits/48d3037e3c39c1db66a2ca449728c2991ec19fc2

**Review by Maffooch — APPROVED** [review]

**Pull request merged**